Dockerfile

```diff
@@ -55,12 +72,18 @@ RUN wget -O /opt/docker/update/genesis https://snapshots.helium.wtf/genesis.${BU
 FROM ${RUNNER_IMAGE} as runner
 
 ARG VERSION
-ARG EXTRA_RUNNER_APK_PACKAGES
-
-RUN apk add --no-cache --update ncurses dbus libsodium libstdc++ \
-                                ${EXTRA_RUNNER_APK_PACKAGES}
-
-RUN ulimit -n 64000
+ARG EXTRA_RUNNER_APT_PACKAGES
+
+RUN apt update \
+    && apt install -y \
+       dbus \
+       iproute2 \
+       libncurses6 \
+       libsodium23 \
+       libstdc++6 \
+       ${EXTRA_RUNNER_APT_PACKAGES} \
+    && rm -rf /var/lib/apt/lists \
+    && ln -sf /opt/miner/releases/${VERSION} /config
```

Comment:
not sure this symlink statement will succeed until that `/opt/miner` is copied into from the builder copy statement below

Reply:
you would think that confirming the target exists is something that `ln` does, however it doesn't seem to. so, for the benefit of reducing a layer in the final image, i moved this up into the other run command. could be easily moved back too if you think that is safer.